packages/frontend-2/components/viewer/comments/Editor.vue

```diff
@@ -1,6 +1,8 @@
 <!-- eslint-disable vuejs-accessibility/no-autofocus -->
 <template>
-  <div class="flex flex-col w-full max-h-28 overflow-y-auto simple-scrollbar">
+  <div
```

Comment:
double check that everything is responsive w/ working scrollbars on various screen sizes

Reply:
I checked on mobile/tablet and by just resizing my screen, anything else in particular you usually test?

Reply:
good enough imo